packages/web-core/src/services/ConnectService.ts

```diff
@@ -314,6 +309,7 @@ export class ConnectService {
   async startAppend(
     appendTokenValue: string,
     abortController?: AbortController,
+    initiatedByUser?: boolean,
```

Comment:
this indicates if the append was started by the user (i.e. from passkey-list) or if it was offered by the component (i.e. after a login)